src/Microsoft.ML.Data/DataLoadSave/TransformWrapper.cs

```diff
@@ -132,19 +72,8 @@ private static bool IsChainRowToRowMapper(IDataView view)
         IRowToRowMapper ITransformer.GetRowToRowMapper(DataViewSchema inputSchema)
         {
             _host.CheckValue(inputSchema, nameof(inputSchema));
-            var input = new EmptyDataView(_host, inputSchema);
-            var revMaps = new List<IRowToRowMapper>();
-            IDataView chain;
-            for (chain = ApplyTransformUtils.ApplyAllTransformsToData(_host, _xf, input); chain is IDataTransform xf; chain = xf.Source)
-            {
-                // Everything in the chain ought to be a row mapper.
-                _host.Assert(xf is IRowToRowMapper);
-                revMaps.Add((IRowToRowMapper)xf);
-            }
-            // The walkback should have ended at the input.
-            Contracts.Assert(chain == input);
-            revMaps.Reverse();
-            return new CompositeRowToRowMapper(inputSchema, revMaps.ToArray());
+            return new CompositeRowToRowMapper(inputSchema,
+                new[] { (IRowToRowMapper)ApplyTransformUtils.ApplyTransformToData(_host, (IDataTransform)_xf, new EmptyDataView(_host, inputSchema)) });
```

Comment:
>IRowToRowMapper [](start = 25, length = 15)

Check that the result of `ApplyTransformToData` is indeed an `IRowToRowMapper`.
(I think that this checked by checking that `_isRowToRowMapper` is true)